packages/cmdliner/cmdliner.0.9.7/opam

```diff
@@ -0,0 +1,16 @@
+opam-version: "1.2"
+maintainer: "Daniel Bünzli <daniel.buenzl i@erratique.ch>"
+authors: ["Daniel Bünzli <daniel.buenzli i@erratique.ch>"]
```

Comment:
Thanks sharp eyes ! Will correct upstream (have to bulk correct that in my repos...). But I think you can merge it as is.